### PR TITLE
OPS: Update RN

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tumvnft3wyafol2wjoe666776e?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11240)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dkilss6mxavqg2n6ujjjhjx4na?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11241)

**Comment by Overtorment** [comment]
Tests failed

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/5ew3pgc2kd7n7lbtpdt7wmhvfu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11253)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: package-lock.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
         "prop-types": "15.8.1",
         "react": "18.2.0",
         "react-localization": "github:BlueWallet/react-localization#ae7969a",
-        "react-native": "0.71.6",
+        "react-native": "^0.71.7",
```
Comment (Overtorment) [inline package-lock.json:61]: Not pinned

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "@types/create-hash": "^1.2.2",
     "@types/jest": "^29.4.0",
     "@types/react": "^18.0.24",
-    "@types/react-native": "^0.71.5",
+    "@types/react-native": "^0.71.6",
```
Comment (Overtorment) [inline package.json:19]: Not pinned

**Review by Overtorment — CHANGES_REQUESTED** [review]
Tests failed

File: package.json (hunk 2 above)
Comment (marcosrdz) [reply package.json:19]: devDependency. none are pinned

**Comment by Overtorment** [comment]
builds failing

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/n4d24um4kpahymr3fxzmch6xgy?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11262)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**